core-bundle/src/Exception/ServiceUnavailableException.php

```diff
@@ -12,12 +12,8 @@
 
 namespace Contao\CoreBundle\Exception;
 
-trigger_deprecation('contao/core-bundle', '4.1', 'Using the "Contao\CoreBundle\Exception\ServiceUnavailableException" class has been deprecated and will no longer work in Contao 5.0. Use the "Lexik\Bundle\MaintenanceBundle\Exception\ServiceUnavailableException" class instead.');
+use Symfony\Component\HttpKernel\Exception\ServiceUnavailableHttpException;
 
-/**
- * @deprecated Deprecated since Contao 4.1, to be removed in Contao 5.0; use the
- *             Lexik\Bundle\MaintenanceBundle\Exception\ServiceUnavailableException instead
- */
-class ServiceUnavailableException extends \RuntimeException
+class ServiceUnavailableException extends ServiceUnavailableHttpException
```

Comment:
Are you using this class anywhere?

Reply:
I need to bring it back because you should use this one instead of the lexik one. We're using it in the PrettyErrorScreenListener. But once https://github.com/contao/contao/pull/3729 is merged (which has to happen first), that will already be the case. So any remarks regarding this exception should be done there :)

Reply:
FYI undeprecating this is also done by https://github.com/contao/contao/pull/3729, but I don't think we should change the parent method. We don't extend `NotFoundHttpException` in our `PageNotFoundException` either etc. Our `ExceptionConverterListener` will correctly handle and convert the exception.

Reply:
#3729 has been merged meanwhile, so this actually needs to be reverted, right?